src/index.js

```diff
@@ -0,0 +1,151 @@
+    // state city name from
+    city: "Seattle",
+    temperature: 0, 
+    lat: 0,
+    lon: 0,
+    original_landscape: "🌵🌼🌸🍁🌵🌼🌸🍁"
+};
+
+const getCityName = (event) => {
+    const inputElement = document.getElementById("city-name")
+    state.city = inputElement.value;
+    const cityDisplay = document.querySelector("h3");
+    cityDisplay.textContent = "For the lovely city of " + "🌟" + state.city + "🌟";
+}
+
+const increaseTemp = (event) => {
+    state.temperature ++;
+    const tempCount = document.getElementById("temp-count");
+    tempCount.textContent = `${state.temperature}`;  
+    changeColorByTemp();
+};
+
+const decreaseTemp = (event) => {
+    state.temperature -= 1;
+    const tempCount = document.getElementById("temp-count");
+    tempCount.textContent = `${state.temperature}`;  
+    changeColorByTemp();
+};
+
+const resetLocation = (event) => {
+    const locationInput = document.getElementById("city-name");
+    locationInput.value = "";
+    state.city = "";
+    getCityName(); //call getCityName to reset h3
+    state.temperature = 0;
+    const tempCount = document.getElementById("temp-count");
+    tempCount.textContent = state.temperature;
+    changeColorByTemp();
+    const emojiLandscape = document.querySelector('#landscape');
+    emojiLandscape.textContent = state.original_landscape;
+    
+};
+
+const changeSky = (event) => {
+    const selectElement = document.querySelector(".sky-select");
+    const output = selectElement.value;
+    const sky = document.querySelector("#sky")
+    if (output === 'sunny') {
+        sky.textContent = '☁️ ☁️ __❗ 🌞❗__   ☁️☁️';
+    } else if (output === 'cloudy') {
+        sky.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+    } else if (output === 'rainy') {
+        sky.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+    } else if (output === 'snowy') {
+        sky.textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+    }
+}
+
+const changeColorByTemp = (event) => {
+    // make js to change color using state temperature variable
+    const tempCount = document.querySelector('#temp-count');
+    const emojiLandscape = document.querySelector('#landscape')
+    if (state.temperature >= 80) {
+        tempCount.setAttribute("class", "red");
+        emojiLandscape.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+    } else if (state.temperature >= 70 && state.temperature <= 79) {
+        tempCount.setAttribute("class", "orange");
+        emojiLandscape.textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+    } else if (state.temperature >= 60 && state.temperature <= 69) {
+        tempCount.setAttribute("class", "yellow");
+        emojiLandscape.textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃"
+    } else if (state.temperature >= 50 && state.temperature <= 59) {
+        tempCount.setAttribute("class", "green");
+        emojiLandscape.textContent= "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    } else if (state.temperature <= 49) {
+        tempCount.setAttribute("class", "teal");
+        emojiLandscape.textContent= "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    }
```

Comment:
Nice if/else logic! It might be worth pulling out the `emojiLandscape` strings into an object for code cleanliness /  we can easily add more options as climate change continues to expand our horizons 🥵